2017-10-22T19:09:59.366Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'id:c4ca4238a0b923820dcc509a6f75849b' in collection with ID 'Participant:org.aabo.Client' does not exist)
    at /Users/a01334390/Documents/Git Projects/Lyra/node_modules/fabric-client/node_modules/grpc/src/node/src/client.js:434:17 {}$
2017-10-22T19:09:59.366Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'id:c4ca4238a0b923820dcc509a6f75849b' in collection with ID 'Participant:org.aabo.Client' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'id:c4ca4238a0b923820dcc509a6f75849b' in collection with ID 'Participant:org.aabo.Client' does not exist)\n    at channel.queryByChaincode.then.catch (/Users/a01334390/Documents/Git Projects/Lyra/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:749:34)\n    at <anonymous>"}$
